### PR TITLE
Feature/300-model-update-pcf

### LINKED ISSUE #300: [Model Update]: PCF
## Update Reason
After release of the latest PCF data model v4.0.1 there were some change requests which need to be implemented in an updated version (e.g. missing properties, adaptation of mandatory/ optional declarations, etc.). All findings are documented in CX confluence.

## MS1 Criteria

- [ ] The model that should be updated exists
- [ ] The referenced use case exists
- [ ] The potential updates are discussed with all stakeholders
- [ ] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [ ] The potential update will be either 
  - [ ] backward compatible or 
  - [ ] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed

### PR DESCRIPTION
**Pull request by SMaierTSI**
## Description
Adapt existing PCF (Product Carbon Footprint) model as aligned with the Catena-X PCF team and the implementation partners. Changes include
- migration from BAMM to SAMM
- additional property "distributionStageAircraftGhgEmissions"
- declaration change of properties "productOrSectorSpecificRules" and "geographyRegionOrSubregion" from optional to mandatory

Closes #300 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [ ] units are referenced from the BAMM unit catalog whenever possible
- [ ] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [ ] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by SMaierTSI** [comment]
@Modeling_Team
@agg3fe 
@bs-jokri

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by agg3fe** [comment]
@SMaierTSI  we can keep this PR as it is until we need it again.

**Comment by SMaierTSI** [comment]
@agg3fe : please check again, see comment in latest commit for details what has changed

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 1]
```diff
@@ -0,0 +1,903 @@
...
+   samm:preferredName "PCF (Product Carbon Footprint) Legal Statement"@en ;
+   samm:description "Optional: Option for legal statement/ disclaimer as specified in the Catena-X PCF Rulebook."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "This PCF (Product Carbon Footprint) is for information purposes only. It is based upon the standards mentioned above." .
+
+:IdTrait a samm-c:Trait ;
+   samm:preferredName "ID (Identifier) Trait"@en ;
+   samm:description "Trait for defining a PCF (Product Carbon Footprint) identifier as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm-c:baseCharacteristic :PfId ;
+   samm-c:constraint :UUIDv4 .
```
Comment (agg3fe) [inline io.catenax.pcf/5.0.0/Pcf.ttl:161]: Here you can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR45

File: io.catenax.pcf/RELEASE_NOTES.md [hunk 2]
```diff
@@ -3,6 +3,13 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [5.0.0] 2023-12-11
```
Comment (agg3fe) [inline io.catenax.pcf/RELEASE_NOTES.md:6]: you can change the date to 4th Dec if you want MS3 in next weekly.

Comment (SMaierTSI) [reply io.catenax.pcf/RELEASE_NOTES.md:6]: Unfortunately i won't be able to implement the change that you proposed before Dec 04th, so Dec 11th should be ok from my perspective. Would be great if we could achieve MS2 and MS3 then with regards to the upcoming holidays

File: io.catenax.pcf/5.0.0/Pcf.ttl (hunk 1 above)
Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: @agg3fe ; @jacewski-bosch : changed to use existing shared characteristic and removed redundant traits/ constraints. Please check again. As already mentioned i would like to achieve MS2 and MS3 during our next regular call on Dec 11th 2023. Thanks in advance.

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 3]
```diff
@@ -0,0 +1,822 @@
...
+
+:crossSectoralStandardsUsed a samm:Property ;
+   samm:preferredName "Cross Sectoral Standards Used"@en ;
+   samm:description "Mandatory: Discloses the cross-sectoral standards applied for calculating or allocating GHG (Greenhouse Gas) emissions as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :CrossSectoralStandardSet .
+
+:productOrSectorSpecificRules a samm:Property ;
+   samm:preferredName "Product or Sector Specific Rules"@en ;
+   samm:description "Mandatory: Product or sector specific rules applied for calculating or allocating GHG (Greenhouse Gas) emissions, e.g. PCRs (Product Category Rules), including operators or pubishers and according rule names as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
```
Comment (catroest) [inline io.catenax.pcf/5.0.0/Pcf.ttl:311]: pubishers --> publishers?

**Review by catroest — COMMENTED** [review]
just some typos @SMaierTSI

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 4]
```diff
@@ -0,0 +1,822 @@
...
+
+:PartialFullPcfCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Partial Or Full PCF (Product Carbon Footprint) Characteristic"@en ;
+   samm:description "Characteristic for defining an indicator for partial or full PCF (Product Carbon Footprint) declaration as specified in the Catena-X PCF Rulebook."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Cradle-to-gate" "Cradle-to-grave" ) .
+
+:PrecedingPfIdsCharacteristic a samm-c:List ;
+   samm:preferredName "Previous PCF (Product Carbon Footprint) Identifiers Characteristic"@en ;
+   samm:description "Caracteristic for defining a non-empty set of product (carbon) footprint identifiers as specified in the Catena-X PCF Rulebook in accordance with the WBCSD (World Business Council for Sustainable Development) Pathfinder framework and the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD/ PACT initiative."@en ;
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: Fixed, thank you for reviewing

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 5]
```diff
@@ -0,0 +1,822 @@
...
+   samm:preferredName "Percent Constraint"@en ;
+   samm:description "Constraint for a decimal number in the range of and including 0 and 100."@en ;
+   samm-c:minValue "0.0"^^xsd:decimal ;
+   samm-c:maxValue "100.0"^^xsd:decimal ;
+   samm-c:lowerBoundDefinition samm-c:AT_LEAST ;
+   samm-c:upperBoundDefinition samm-c:AT_MOST .
+
+:EmissionFactorDS a samm:Entity ;
+   samm:preferredName "Emission Factor Data Source Entity"@en ;
+   samm:description "Entitiy for defining an emission factor data source used to calculate a product carbon footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: fixed typo, thanks for review

File: io.catenax.pcf/5.0.0/Pcf.ttl (hunk 3 above)
Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: fixed typo, thanks for review

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 6]
```diff
@@ -0,0 +1,822 @@
...
+
+:EmissionFactorDSSet a samm-c:Set ;
+   samm:preferredName "Emission Factor Data Sources Set Characteristic"@en ;
+   samm:description "Characteristic for defining a set of emission factor sources used for calculating a product carbon footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:dataType :EmissionFactorDS .
+
+:DataQualityIndicators a samm:Characteristic ;
+   samm:preferredName "Data Quality Indicators"@en ;
+   samm:description "Characteristic for defining the quantative data quality indicators of a PCF (Product Carbon Footprint) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
```
[the remark this answers is not in the document]
Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: fixed typo, thanks for review

File: io.catenax.pcf/5.0.0/Pcf.ttl [hunk 7]
```diff
@@ -0,0 +1,822 @@
+#######################################################################
+# Copyright (c) 2022, 2023 ZF Friedrichshafen AG
+# Copyright (c) 2022, 2023 T-Systems International GmbH
+# Copyright (c) 2022, 2023 Robert Bosch Manufacturing Solutions GmbH
+# Copyright (c) 2023 Siemens AG
+# Copyright (c) 2023 SAP AG
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
```
Comment (agg3fe) [inline io.catenax.pcf/5.0.0/Pcf.ttl:7]: you can remove 2022

Comment (SMaierTSI) [reply io.catenax.pcf/5.0.0/Pcf.ttl]: Done

**Comment by jacewski-bosch** [comment]
All required reviewers have approved this PR (see reviewers section) --> OK
 The new aspect (version) will be implemented by at least one data provider --> OK
 The new aspect (version) will be consumed by at least one data consumer --> OK
 There exists valid test data --> OK
 In case of a new (incompatible) major version to an existing version, a migration strategy has been developed --> OK
 The model has at least version '1.0.0' --> OK
 The release date in the Release Note is set to the date of the MS3 approval --> OK

 MS3 approved by group

@agg3fe please merge

**Pull request merged**